scripts/dev-setup/main.py

```diff
@@ -0,0 +1,138 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import argparse
+import os
+import site
+import typing
+from dataclasses import dataclass
+from pathlib import Path
+
+from test_resources import REGISTERED_TEST_RESOURCES_FUNCTIONS
+from utils import REPO_ROOT_DIR, change_cwd, print_blue, run_cmd
+
+PROMPT_FLOW_PKGS = [
+    "promptflow-tracing",
```

Comment:
how do promptflow-tools do dev setup now?

Reply:
tools has separate dev setup for now: <https://github.com/microsoft/promptflow/blob/main/src/promptflow-tools/README.dev.md>, so this script does not support tool for now

Reply:
if needed, i can add tools setup here